### ISSUE TITLE
support session env vars

### ISSUE THREAD

**Issue by danehammer** [aws/amazon-ecs-cli#48]
If my docker-compose.yml specifies an env var without a value, docker compose uses the shell you're running in to grab that value and pass it along. This is useful for not source controlling things like API tokens. E.g.

```yml
hubot:
  image: danehammer/the_dude
  links:
    - redis
  environment:
    - "REDIS_URL=redis://redis:6379"
    - "HUBOT_SLACK_TOKEN"
```

This allows me to export the token as a var locally when I go to run the compose up command. Works correctly with docker compose, does not work with ecs-cli compose.

**Comment by samuelkarp** [comment]
Similar to #28, but sufficiently different that this can remain open.

**Comment by synotna** [comment]
Much needed

In the meantime, my workaround is
    sed 's/FOO/FOO=$BAR/g' docker-compose.yml

**Comment by uttarasridhar** [comment]
released in [v0.3.0](https://github.com/aws/amazon-ecs-cli/releases/tag/v0.3.0)
